microsoft-edge/toc.yml

```
@@ -1033,89 +1050,166 @@
               href: extensions-chromium/developer-guide/sidebar.md
               displayName: sidebar extensions, sidepanel, side panel  # alt lookup phrase
 
-            - name: Best Practices
-              href: extensions-chromium/developer-guide/best-practices.md
-              displayName: Best Practices for extensions  # full title
+            - name: Manifest files
+              items:
+                - name: Manifest file format for extensions
+                  href: extensions-chromium/getting-started/manifest-format.md
 
-        - name: Manifest
-          items:
-            - name: Overview and timelines for migrating to Manifest V3
-              href: extensions-chromium/developer-guide/manifest-v3.md
+                - name: Using Content Security Policy (CSP) to control which resources can be run by an extension
+                  href: extensions-chromium/developer-guide/csp.md
+                  displayName: policies  # synonym lookup
 
-            - name: Migrate an extension from Manifest V2 to V3
-              href: extensions-chromium/developer-guide/migrate-your-extension-from-manifest-v2-to-v3.md
+                - name: Overview and timelines for migrating to Manifest V3  # overview of Manifest V3, and timeline for migrating
+                  href: extensions-chromium/developer-guide/manifest-v3.md
 
-            - name: Manifest file format for extensions
-              href: extensions-chromium/getting-started/manifest-format.md
+                - name: Migrate an extension from Manifest V2 to V3
+                  href: extensions-chromium/developer-guide/migrate-your-extension-from-manifest-v2-to-v3.md
 
-        - name: Publish to the Add-ons website
+            - name: JavaScript APIs
+              items:
+                - name: Microsoft Edge Add-ons API 101
+                  href: extensions-chromium/developer-guide/apis-intro.md
+
+                - name: Supported APIs for Microsoft Edge extensions
+                  href: extensions-chromium/developer-guide/api-support.md
+
+                - name: Declare API permissions in extension manifests
```

```suggestion
                - name: Declare API permissions in the manifest
```
prefer singular (concrete) over plural (abstract) - article actually discusses singular "in the extension's manifest", not "in your manifests"

change in all the usual spots

---

common improvement pattern: 
changing a title from plural to singular = 
changing from abstract, _system-oriented doc'n_, to concrete, _task-oriented doc'n_.  

a giveaway/clue is that the body of article typically is worded in the singular, contradicted by title that's mis-represented as plural.

a task or use-case is typically singular, not plural.

@captainbrosset